### ISSUE TITLE
UH1 - Windscreen

### ISSUE THREAD

**Issue by CandidYams** [sw4l/15thsoc#76]
New 'glass' for the UH1 windscreen produces an amazing amount of glare and reflections that can make it difficult to navigate obstacles in low altitude flight.

Compared to this new windscreen, I prefer its dirtier but much less reflective predecessor. Such is the severity of the reflections and glare. If it's possible to implement a clean windscreen without the glare and reflections, that would be wonderful.

Respectfully,

2ndLt J. Leonard
Gunfighter 1-3 AC
HMLA-369 UH-1

**Comment by EvanWeeks** [comment]
This is definitely an issue. Last night in UH-1 training, any strong light source within 100m of the aircraft, *even the aircraft's own spotlight,* created a glare on the windscreen that obscured everything. If using NVGs, this washed out the entire view. Anytime we were anywhere near the Altis airfield, the windscreen was a mess of glares that were making seeing anything around us a nightmare. It also murdered framerate.

1stLt Weeks
Gunfighter 1-2 AC
HMLA-369 UH-1

**Comment by Chewy1007** [comment]
I would appreciate it if we could flag this as a priority issue for UH1. 

If we were currently needed for a nighttime op right now, I'm not sure I would allow these birds in the air. The windshield makes it impossible to see anything in the presence of bright lights (ie. using the spotlight at an airfield/fleet, or in the event of weapons deployment), and is nearly impossible to see through using NVGs. 

Your assistance on this matter would be greatly appreciated. Thank you, 

respectfully,

Capt C. C. Watson
Detachment Commander
HMLA-369 "Gunfighters" (UH1Y)

**Comment by Karneck** [comment]
I'll revert the change for the next modpack update

**Comment by Karneck** [comment]
Old glass restored.

**Comment by EvanWeeks** [comment]
Old glass has apparently not been restored. Or, rather, it has... it just coexists with the reflective glass. We got frustrated with the glass the other day and decided to shoot it out of the UH-1Y during training. What we discovered is that the OUTER pane (removed by a three-round burst of 5.56) is reflective, while the INNER pane is the old glass. If we remove the outer pane via the 5.56 method, we have the old glass back.

I would prefer to not have my pilots use this method of self-customizing their aircraft. It breaks immersion, to say the least, and violates a number of unit policies. The alternative, of course, is to ground our aircraft for night ops until this is fixed.

1stLt Evan Weeks
Detachment Commander
HMLA-369 "Gunfighters" (UH1Y)
